test/e2e/experiment_test.go

```diff
@@ -99,6 +99,23 @@ func (s *ExperimentSuite) TestExperimentWithServiceAndScaleDownDelay() {
 		ExpectExperimentServiceCount("experiment-with-service", 0)
 }
 
+func (s *ExperimentSuite) TestExperimentWithMultiportServiceAndScaleDownDelay() {
+	g := s.Given()
+	g.ApplyManifests("@functional/experiment-with-multiport-service.yaml")
+	g.When().
+		WaitForExperimentPhase("experiment-with-multiport-service", "Running").
+		Sleep(time.Second*5).
+		Then().
+		ExpectExperimentTemplateReplicaSetNumReplicas("experiment-with-multiport-service", "test", 1).
+		ExpectExperimentServiceCount("experiment-with-multiport-service", 1).
+		When().
+		WaitForExperimentPhase("experiment-with-multiport-service", "Successful").
+		Sleep(time.Second*15).
```

Comment:
```suggestion
```suggestion
                WaitForExperimentCondition("experiment-with-service", func(ex *rov1.Experiment) bool {
			return s.GetReplicaSetFromExperiment(ex, "test").Status.Replicas == 0
		}, "number-of-rs-pods-meet", fixtures.E2EWaitTimeout).
```
```